### ISSUE TITLE
Disabling access to private repositories

### ISSUE THREAD

**Issue by pavel-zhigulin** [microsoft/vscode-remote-repositories-github#14]
When you install extension it asks permission to private repositories, but I don't want to grant this access. 

It would be great if you'll be able to restrict extention to be able to search by public repos only.

**Comment by eamodio** [comment]
Yeah, we will be looking at supporting this scenario in a future release (not sure when yet though).

**Comment by scscgit** [comment]
What a great way to discourage anyone capable of common sense from ever trying out this extension. It's as if its designers don't realize how irresponsible is it to request a complete access to someone's GitHub account and all their repositories just because they want to open a read-only version of some public repository. Sure, let's ignore the lack of granularity to be able to approve just individual requests, e.g. interactions with pull requests, and assume that we want any future UX failures, bugs, and security breaches to do random actions on our behalf. I'm sure all users properly "sign out from their VS Code" after they use it on any shared computer. With this trend, the future sure looks bright for all the hackers if Microsoft developers openly embrace this mindset of an absolute trust in all extensions; and what's the better way to develop this habit if not providing a full access even for every trivial action of downloading something that's public? :)
